rsconnect/actions.py

```diff
@@ -279,6 +279,8 @@ def deploy_jupyter_notebook(connect_server, file_name, extra_files, new=False, a
     app_id, deployment_name, deployment_title, app_mode = \
         gather_basic_deployment_info(connect_server, app_store, file_name, new, app_id, title, static)
     python, environment = get_python_env_info(file_name, python, compatibility_mode, force_generate)
+    if environment.get('error', None) is not None:
```

Comment:
```suggestion
    if 'error' in environment:
```

Your suggestion anyway and it reads a lot cleaner.